package/yast2.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Jul 28 10:50:05 UTC 2015 - ancor@suse.com
+
+- Properly display error information when running in command line
```

Comment:
Well, actually it hides everything reported by means of `Report.Error`. That includes exceptions (in the current implementation of WFM) but also more things. Other than that, I find "error" to be a more end-user friendly word than "exception".